### PR TITLE
Slug 1.0.1

### PR DESCRIPTION
**Pull request by thangngoc89**
Patch releases that makes Slug's behavior consistent with node-slugify.
String is normalized with NFC instead of NFD

### CODE REVIEW

File: packages/slug/slug.1.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+bug-reports: "https://github.com/thangngoc89/ocaml-slug/issues"
+license: "MIT"
+build: [
+  [ "dune" "subst" ] {dev}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" {>= "2.0"}
+  "opam-lock" {dev}
```
Comment (kit-ty-kate) [inline packages/slug/slug.1.0.1/opam:15]: why is opam-lock required in dev mode? I can’t find any mention of it in the source code

**Comment by kit-ty-kate** [comment]
The failures can be ignored btw.

File: packages/slug/slug.1.0.1/opam (hunk 1 above)
Comment (thangngoc89) [reply packages/slug/slug.1.0.1/opam:15]: I have no ideas. I copied this from 1.0.0. Let’s me delete this

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the fix in the opam file upstream so that it can be taken into account for next time?

**Comment by thangngoc89** [comment]
Sure. Thank you very much

**Pull request merged**